service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/controller/ReportController.java

```diff
@@ -46,23 +55,58 @@
         produces = TEXT_XML_VALUE)
     @ResponseStatus(value = ACCEPTED)
     public ResponseEntity<String> postReport(@RequestBody String reportXml) {
+        String toAddress = null;
```

Comment:
` = null;` is not needed I believe. :P

Reply:
Is needed for Exception catch

Reply:
I mean `null` is default value for string anyway. ;)